internal/scorecard/alpha/run_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func TestFakeRunner(t *testing.T) {
```

Comment:
Nit: we're testing `o.RunTests` here right? We're just _using_ the fake runner to do that.

Suggest renaming to:
```suggestion
func TestRunTests(t *testing.T) {
```

Reply:
changed the name as suggested.